examples/outliers/alibi-detect-combiner/pipeline/combiner/Combiner.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import json
+import numpy as np
+
+class Combiner(object):
+
+    def aggregate(self, X, features_names=[]):
+        logging.warning(X)
+        output = {
+            "loanclassifier": X[0],
+            "outliersdetector": json.loads(X[1]),
+        }
+        return json.dumps(output, cls=NumpyEncoder)
```

Comment:
This would return a string so it would use strData (Which is how the XAI example works atm), but it may be best if you return it as a dictionary instead so it uses the jsonData

Reply:
Sure, will try and see how it works. I was not sure if it will work so I just did what some other example did -> I guess there is a problem if input contains numpy arrays as one of values?

Reply:
Done in https://github.com/SeldonIO/seldon-core/pull/1306/commits/a2839d4b63e09b138d18b549fef538583dcafec8.